frontend/src/components/common/Platform/Dialog/DeletePlatform.vue

```diff
@@ -71,7 +71,7 @@ function closeDialog() {
           >{{ platform.name }} - [<span class="text-romm-accent-1">{{
             platform.fs_slug
           }}</span
-          >] from RomM.</span
+          >].</span
```

Comment:
shouldnt we keep "from romm" so users know it's not getting removed from the fs?

Reply:
Hmm good point, I thought it was too repetitive, but I can readd it